helpers/stylesheet.js

```diff
@@ -18,12 +19,19 @@ const factory = globals => {
 
         const url = cdnify(path);
 
-        let attrs = { rel: 'stylesheet' };
-
+        if (_.has(options.hash, 'resourceHint')) {
+            addResourceHint(
+                globals,
+                url,
+                options.hash['resourceHint'],
+                resourceHintAllowedTypes.resourceHintStyleType
+            );
+            delete options.hash.resourceHint;
+        }
+
+        let attrs = {rel: 'stylesheet'};
         Object.assign(attrs, options.hash);
-
-        attrs = _.map(attrs, (value, key) => `${key}="${value}"`).join( ' ');
-
+        attrs = _.map(attrs, (value, key) => `${key}="${value}"`).join(' ');
```

Comment:
you can try to use `for.. in` here, and I also don't recommend to override `attrs` variable